app/assets/javascripts/Components/Modals/send_lti_grades_modal.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import Modal from "react-modal";
+
+class LtiGradeModal extends React.Component {
+  static defaultProps = {
+    override: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      deploymentsChecked: undefined,
+    };
+  }
+
+  componentDidMount() {
+    Modal.setAppElement("body");
+    const deploymentsMapped = this.props.lti_deployments.map(deployment => ({
```

Comment:
Please move this logic to the constructor and set the initial state value for the `deploymentsChecked` key instead of setting it here. 

This can cause an unnecessary re-render otherwise.